<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-4RCGPP9C" failures="1" tests="3" name="Test1" time="56.324" errors="0" timestamp="2024-03-21T09:11:44 IST">
  <testcase classname="Com.Ecommerce.TestCases.TC_Login_Page" name="LoginPage" time="0.914">
    <skipped/>
  </testcase> <!-- LoginPage -->
  <testcase classname="Com.Ecommerce.TestCases.TC_Login_Page" name="LoginPage" time="4.187">
    <skipped/>
  </testcase> <!-- LoginPage -->
  <testcase classname="Com.Ecommerce.TestCases.TC_Login_Page" name="LoginPage" time="3.411">
    <failure type="java.lang.AssertionError" message="expected [nopCommerce demo store] but found [nopCommerce demo store. Login]">
      <![CDATA[java.lang.AssertionError: expected [nopCommerce demo store] but found [nopCommerce demo store. Login]
at Com.Ecommerce.TestCases.TC_Login_Page.LoginPage(TC_Login_Page.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- LoginPage -->
</testsuite> <!-- Test1 -->
